### ISSUE TITLE
How do you use config.properties to set the test url?

### ISSUE THREAD

**Issue by scottcsims** [partnet/seauto#10]
I set test.config.url= and the tests still went to bing.com in the Junit example.  Is this as designed?  I want to change the url using config.properties.

**Comment by bbarke** [comment]
Looks like this is a bug for all projects. For the time being, add this to your BingSite.java class located at `/src/test/java/<your-project>/page/BingSite.java`

``` java
  /**
   * Open the home page.
   */
  public void open()
  {
    this.open(getDefaultUrl());
  }
```

**Comment by bbarke** [comment]
Should be fixed with merge #11

**Comment by bbarke** [comment]
This is fixed in latest release (0.9.1)
